### PR TITLE
Feat/met 4864 add spring boot support metis authentication

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: metis-authentication/metis-authentication-rest/manifest.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-  host: metis-authentication-rest-test-compose
-  path: target/metis-authentication-rest-1.0-SNAPSHOT.war
-  buildpack: https://github.com/europeana/java-buildpack.git#feat/java8
-  services:
-    - metis-authentication-test-psql-compose
+  - path: target/metis-authentication-rest-9-SNAPSHOT.jar
+    name: metis-authentication-rest-test
+    host: metis-authentication-rest-test
+    stack: cflinuxfs3
+    buildpack: https://github.com/cloudfoundry/java-buildpack
```
Comment (jeortizquan) [inline metis-authentication/metis-authentication-rest/manifest.yml:7]: Double check if this apply: 
according to documentation this attribute `buildpack` is deprecated. They favor the use of `buildpacks`
example
```
   buildpacks:
   - https://github.com/cloudfoundry/java-buildpack
```

**Review by jeortizquan — APPROVED** [review]
It looks good, runs independent locally. Just have questions of the manifest file.

File: metis-authentication/metis-authentication-rest/manifest.yml (hunk 1 above)
Comment (stzanakis) [reply metis-authentication/metis-authentication-rest/manifest.yml:7]: Good point, I didn't know.
But I'll leave it as is for now, we will migrate very soon anyway.

File: metis-authentication/metis-authentication-rest/manifest.yml [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
-- name: metis-authentication-rest-test-compose
-  memory: 2048M
-  instances: 1
-  host: metis-authentication-rest-test-compose
-  path: target/metis-authentication-rest-1.0-SNAPSHOT.war
-  buildpack: https://github.com/europeana/java-buildpack.git#feat/java8
-  services:
-    - metis-authentication-test-psql-compose
+  - path: target/metis-authentication-rest-9-SNAPSHOT.jar
+    name: metis-authentication-rest-test
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-authentication/metis-authentication-rest/manifest.yml]: It's supposed to be a sample.
I removed the test prefix to avoid confusion.

**Review by jeortizquan — APPROVED** [review]

**Pull request merged**